smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -105,6 +105,17 @@ function displayReaderList(readers) {
     var indicatorContainer = goog.dom.createDom(
         'span', 'reader-indicator-container', indicator);
 
+    var toolTipText = '';
+    if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.INIT) {
```

Comment:
nit: I think it's a good case where "switch" can be used. Also perhaps let's extract this into a separate function, so that the displayReaderList() function doesn't become overly long.

A "switch" is more readable and less error-prone when we need to exhaustively check all possible values of an enum.